README.md

```diff
@@ -1 +1,146 @@
-# amazon-dash-buttons
\ No newline at end of file
+## Amazon-Dash-Buttons
+
+A docker image for listening on Amazon Dash Buttons and trigger events when one gets clicked.
+
+You can get the build docker image form here:
+
+[Docker Hub](https://hub.docker.com/r/fusonic/amazon-dash-buttons/)
+
+### Prerequisities
+
+
+In order to run this container you'll need docker installed.
+
+* [Windows](https://docs.docker.com/windows/started)
+* [OS X](https://docs.docker.com/mac/started/)
+* [Linux](https://docs.docker.com/linux/started/)
+
+### Usage
+
+#### Run Command
+
+You will have to configure a config.json to configure your Amazon Dash Buttons and give information to services. 
+
+To start the docker container use the following command: 
+
+```bash
+docker run -d --name dash-button --net=host --restart unless-stopped --privileged -v /pathToConfig.json:/config.json fusonic/amazon-dash-buttons:latest
+```
+
+Change pathToConfig.json to match your config.json file.
+
+##### Flags needed due to:
+
+* `--net` - have access to the network, which dash-buttons are connected to
+* `--privileged` - being able to inspect packages in the host system
+* `-v` - mount the json file into container
+
+#### Create config.json
+
+There is a example.config.json located in the root of the project, where you can see how your config file could look like.
```

Comment:
Please insert links to the files in the repository to allow quicker navigation:

https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files